tests/Functional/starWarsSchema.php

```diff
@@ -22,21 +22,26 @@ function starWarsSchema()
                 return getDroid($arguments['id']);
             },
         ],
-        'Human' => [
-            'friends' => function ($human) {
+        'Human'     => [
+            'friends'         => function ($human) {
                 return getFriends($human);
             },
             'secretBackstory' => function () {
                 throw new \Exception('secretBackstory is secret.');
             },
         ],
-        'Droid' => [
-            'friends' => function ($droid) {
+        'Droid'     => [
+            'friends'         => function ($droid) {
                 return getFriends($droid);
             },
             'secretBackstory' => function () {
                 throw new \Exception('secretBackstory is secret.');
             },
         ],
+        'Character' => [
```

Comment:
This wasn't here before, should it be?

Reply:
Yes, because I removed the automatic resolving using the `type` field as it's not part of the spec.